docs/README.md

```diff
@@ -276,6 +287,15 @@ $approval     = $modification->disapprovals()->get();
 $author       = $approval->disapprover();
 ```
 
+## Retrieving disapproval Reason
+
+```php
+$post         = Post::find(1);
+$modification = $post->modifications()->first();
+$approval     = $modification->disapprovals()->get();
+$reason       = $approval->reason;
```

Comment:
Right off the bat without testing, how would this work? `->get()` returns a collection, calling `->reason` on a collection would most likely always return null.

Reply:
I would use `->first()`